### DISCUSSION TITLE
Are there any vulnerabilities with the ATSPM code related to Log4j?

### DISCUSSION THREAD

**Discussion by ZachParkerCR** [udotdevelopment/ATSPM#111] [Q&A] [answered]
Can any of the developers speak to the vulnerabilities that are present related to the Log4j zero-day flaw?  Do we need to update anything on our servers to protect our environment?

**Comment by shanejohnson-atspm** [comment, marked as answer]
We don't use Java or Apache in this project.
I think we are safe from this.
